org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -259,6 +261,10 @@ private void setPreferenceNodeId() {
 	}
 
 	private void configureProxy() {
+		if (Boolean.getBoolean("jdt.ls.proxiesDisabled")) {
```

Comment:
I would call this `jdt.ls.disableProxies` , but looks fine otherwise.

Reply:
Fixed.